package.json

```diff
@@ -34,10 +34,10 @@
   },
   "dependencies": {
     "@edx/brand": "npm:@edx/brand-openedx@1.1.0",
-    "@edx/frontend-component-footer": "11.2.0",
-    "@edx/frontend-component-header": "3.2.0",
-    "@edx/frontend-platform": "2.6.1",
-    "@edx/paragon": "20.15.0",
+    "@edx/frontend-component-footer": "12.0.0",
+    "@edx/frontend-component-header": "4.0.0",
+    "@edx/frontend-platform": "4.2.0",
+    "@edx/paragon": "20.32.0",
```

Comment:
sounds like the plan going forward for these version bumps is to directly use `@openedx` instead of the `@edx` redirect.

Reply:
I think I've missed the memo on these updates 😅.

I tried https://www.npmjs.com/package/@edx/paragon and it's still working, but https://www.npmjs.com/package/@openedx/paragon gives a 404.

I think there's something else I'm missing.

Reply:
sounds like sarina was mistaken about the redirects (npm vs github), keeping them as is works!